net-vpn/protonvpn-cli/protonvpn-cli-2.2.0.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{3_6,3_7} )
+
+inherit distutils-r1
+
+DESCRIPTION="A VPN command-line tool from protonvpn - python rewrite"
+HOMEPAGE="https://protonvpn.com"
+SRC_URI="https://github.com/ProtonVPN/${PN}-ng/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+KEYWORDS="~amd64"
+SLOT=0
+
+RDEPEND="${PYTHON_DEPS}
```

Comment:
`PYTHON_DEPS` is not necessary here. But you are missing few dependencies, 
https://github.com/ProtonVPN/protonvpn-cli-ng/blob/80a4c6db11ccebd3c95cce4b9e5e6be586cca290/requirements.txt